example-conf/mesos-dns/mesos-site.xml

```diff
@@ -43,7 +43,7 @@
 
   <property>
     <name>mesos.hdfs.zkfc.ha.zookeeper.quorum</name>
-    <value>master0.mesos:2181,master1.mesos:2181,master2.mesos:2181</value>
```

Comment:
Talked to @abhay-agarwal, and we will make the default 1 due to the fact that we support clusters with a single Zookeeper. The user can change this variable for production clusters with 3 nodes.